coffee-chats/src/test/java/com/google/step/coffee/PermissionCheckerTest.java

```diff
@@ -13,4 +20,118 @@ public void testEnsureLoggedInWhenLoggedOut() throws Exception {
     helper.setEnvIsLoggedIn(false);
     PermissionChecker.ensureLoggedIn();
   }
+
+  private boolean doGroupTest(Status prev, Status cur) {
```

Comment:
Change method name to something more descriptive, it is not particularly clear. Perhaps `changeGroupStatusSelf`?